install/syndesis.yml

```diff
@@ -1634,6 +1634,14 @@ objects:
     metadata:
       name: syndesis-upgrade-latest
     spec:
+      affinity:
+        podAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchLabels:
+                "syndesis.io/app": "syndesis"
+                "syndesis.io/component": "syndesis-db"
+            topologyKey: "kubernetes.io/hostname"
```

Comment:
Hmm, don't really understand the documentation for this field. Do you have any idea what the topologyKeu is for ?

Reply:
The whole thing means: find a pod with labels matching the ones I provide, then make sure this new pod has the same topology key as the one you've found (if any), i.e. "kubernetes.io/hostname" means same node. See the "redis" example: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#always-co-located-in-the-same-node

Btw I'm going to spin a new PVC for the upgrade pod :)

Reply:
ok. sounds good. But we have not to forget to negotiate this with the OSO team to increase the PV quota for Fuse Online.